tests/test_api.py

```diff
@@ -1,20 +1,11 @@
 from n26 import api, config
-from n26.api import BASE_URL_DE, POST, GET
+from n26.api import POST, GET
 from tests.test_api_base import N26TestBase, mock_auth_token, mock_requests, mock_config
 
 
 class ApiTests(N26TestBase):
     """Common Api tests"""
 
-    def test_create_request_url(self):
```

Comment:
Just removing all tests associated with a given functionality and calling it a day is plain ignorant.
I stated reasons why we use our own custom implementation and maybe we should add tests to this library to make this more obvious. But I am quite sure that our implementation behaves different from the one used by `requests`.